R/cre.R

```diff
@@ -16,24 +16,24 @@
 #'     (default: 0.5).
 #'   - *Parameters for Discovery and Inference*
 #'     - *ite_method*: The method for ITE (pseudo-outcome) estimation
-#'     (default: 'aipw', options: {'aipw' for Augmented Inverse Probability
-#'     Weighting, 'cf' for Causal Forest, 'bart' for Causal Bayesian Additive
-#'     Regression Trees, slearner' for S-Learner, 'tlearner' for T-Learner,
-#'     'xlearner' for X-Learner, 'tpoisson' for T-Learner with Poisson
+#'     (default: "aipw", options: {"aipw" for Augmented Inverse Probability
```

Comment:
Please drop the quotation marks as much as possible. If it is necessary, please use \dQuote. 
https://developer.r-project.org/Rds.html

Reply:
removed